pkg/controllers/provisioning/scheduling/machine.go

```diff
@@ -114,11 +114,6 @@ func (m *Machine) FinalizeScheduling() {
 	delete(m.Requirements, v1.LabelHostname)
 }
 
-func (m *Machine) String() string {
-	return fmt.Sprintf("machine with %d pods requesting %s from types %s", len(m.Pods), resources.String(m.Requests),
```

Comment:
Just to make sure, we're fully removing this because we can just add into the context for the logs, rather than making this the string representation, right? I agree with what you're doing, but just making sure :D

Reply:
Yes, it's being added in the machine controller into structured values